redis/src/commands/mod.rs

```diff
@@ -575,48 +575,48 @@ implement_commands! {
 
     /// Intersect multiple sorted sets and store the resulting sorted set in
     /// a new key using SUM as aggregation function.
-    fn zinterstore<K: ToRedisArgs>(dstkey: K, keys: &'a [K]) {
+    fn zinterstore<D: ToRedisArgs, K: ToRedisArgs>(dstkey: D, keys: &'a [K]) {
```

Comment:
As mentioned above, this command already specifies &[K] for the 2nd type. Doesn't that already get us what we need?

Reply:
No, because the K must be the same for all arguments , so you can't call the command with `dstkey: String` and `keys: &[&str] / &[u8]`.